cmake/Helpers.cmake

```diff
@@ -146,6 +146,16 @@ macro(cvc5_option var description)
   set_property(CACHE ${var} PROPERTY STRINGS IGNORE ON OFF)
 endmacro()
 
+# Wasm options are four-valued to detect if the WebAssembly compilation will
+# happen and if so, find out the extension used in this process.
+# The options are: OFF (not a WebAssembly compilation), WASM (.wasm), JS (.js) 
+# or HTML (.html).
+macro(wasm_option var description)
+  set(${var} IGNORE CACHE STRING "${description}")
+  # Provide drop down menu options in cmake-gui
+  set_property(CACHE ${var} PROPERTY STRINGS OFF WASM JS HTML)
```

Comment:
I wouldn't make this a separate macro, because it only applies to one specific option. Do we do that for any other option?

Reply:
It doesn't happen in any other place. I will fix it.